bw2io/strategies/exiobase.py

```diff
@@ -8,6 +8,40 @@
 
 
 def normalize_units(data, label="unit"):
+    """
+    This function normalizes the units of the given data by replacing them with
```

Comment:
...same comment as: https://github.com/brightway-lca/brightway2-io/pull/175#discussion_r1153681421. Would be great to update grammar for consistency.